libcodechecker/cmd/cmd_line_client.py

```diff
@@ -30,10 +30,7 @@
 
 
 class CmdLineOutputEncoder(json.JSONEncoder):
-    def default(self, obj):
-        d = {}
-        d.update(obj.__dict__)
-        return d
+    pass
```

Comment:
Are you sure we can remove this? [json.JSONEncoder.default](https://docs.python.org/3/library/json.html#json.JSONEncoder.default)